functions/src/api/v1/reminders/createReminder.ts

```diff
@@ -58,15 +61,41 @@ import { assertTimestampInFuture } from './assertTimestampInFuture'
  *       412:
  *         description: authorization header present but not formatted correctly
  */
-export const createReminder = wrap(async (req: Request, res: Response) => {
+export const createReminderRoute = wrap(async (req: Request, res: Response) => {
     const userId = res.locals.uid
     const roomId = req.params.roomId
+    const { hostName, destinations, sendAt } = req.body
+
+    const reminderId = await createReminder({
+        roomId,
+        userId,
+        hostName,
+        sendAtParameter: sendAt,
+        destinationsParameter: destinations,
+    })
+
+    res.send({
+        reminderId,
+    })
+})
+
+export const createReminder = async ({
+    roomId,
+    userId,
+    sendAtParameter,
+    hostName,
+    destinationsParameter,
+}: {
+    roomId: RoomId
+    userId: UID
+    sendAtParameter: string
```

Comment:
as a general rule, I would avoid having to suffix variables with `Parameter` and refactor

Reply:
it conflict with the var inside, how would you name it?

Reply:
seems your handling `seconds` and `millis`, what about `sendAtSecs` & `sendAtMillis` ?